test/Microsoft.AspNetCore.SystemWebAdapters.Framework.Tests/ProxyHeaderModuleTests.cs

```diff
@@ -4,7 +4,7 @@
 using System.Collections.Specialized;
 using Xunit;
 
-namespace Microsoft.AspNetCore.SystemWebAdapters.FrameworkTests;
+namespace Microsoft.AspNetCore.SystemWebAdapters.Tests;
```

Comment:
:)

Reply:
Made it match some of the other framework tests